server.go

```diff
@@ -117,7 +118,7 @@ type server struct {
 // newServer creates a new instance of the server which is to listen using the
 // passed listener address.
 func newServer(listenAddrs []string, chanDB *channeldb.DB, cc *chainControl,
-	privKey *btcec.PrivateKey) (*server, error) {
+	privKey *btcec.PrivateKey, alias string, color color.RGBA) (*server, error) {
```

Comment:
Actually, to not clutter up `lnd_test`, I think we should pass `cfg` to `newServer`, make `parseHex` a method in `server`, and create `server_test.go`. Then the parsing of alias and color can be done in `newServer` instead :)